editor/src/core/shared/runtime-report-logs.tsx

```diff
@@ -154,3 +154,39 @@ export function useErrorOverlayRecords(): ErrorOverlayRecords {
 
   return { errorRecords, overlayErrors }
 }
+
+const ReactRouterErrorPrefix = `React Router caught the following error during render`
+const ReactRouterAwaitErrorPrefix = `<Await> caught the following error during render`
+
+export function listenForReactRouterErrors(targetConsole: Console): void {
+  const targetConsoleAny = targetConsole as any
+
+  // Remove any existing proxy first.
+  if (targetConsoleAny.originalErrorMethod != null) {
+    // Restore the original method.
+    targetConsoleAny.error = targetConsoleAny.originalErrorMethod
+    // Remove this field, thereby restoring this console to its original state.
+    delete targetConsoleAny['originalErrorMethod']
+  }
+
+  // Squirrel away the original method for unpacking later.
+  const originalMethod = targetConsoleAny.error
+  targetConsoleAny.originalErrorMethod = originalMethod
+  targetConsoleAny.error = function (...args: Array<any>) {
+    // Call the original method first.
+    originalMethod(...args)
+
+    // If the first part of the log line is a string
+    const firstLine = args[0]
```

Comment:
Previously we would use `console-feed` to parse, then encode, then decode (?) the `args` (https://github.com/concrete-utopia/utopia/pull/5676/files#diff-e3967556a9487a929c1e232412b7704ffd6fb9d427e4722eb2cb2322963f7c56L55-L57), which creates an object used by that library. We would then check the `data` field of that object, which is just the `args` array (https://github.com/concrete-utopia/utopia/pull/5676/files#diff-a4c8338855ca2b6cba1cde108cc950253be1063727a563185098bbd573c2bea7L134-L145). So since we no longer need to proxy to a console pane, we don't need to do any of that, and can just check the `args` directly.